example/res/xml/payment_sheet_preferences.xml

```diff
@@ -18,9 +18,4 @@
         app:title="Enable Google Pay"
         app:defaultValue="true" />
 
-    <SwitchPreferenceCompat
-        app:key="require_billing_address"
```

Comment:
I think you'll want to remove the places that access the key `require_billing_address` as well